Luminary099/INPUT_OUTPUT_CHANNEL_BIT_DESCRIPTIONS.agc

```diff
@@ -194,7 +194,7 @@
 #		BIT 6		THRUSTERS 13 & 15 DISABLED BY CREW
 #		BIT 7		THRUSTERS 9 & 12 DISABLED BY CREW
 #		BIT 8		THRUSTERS 10 & 11 DISABLED BY CREW
-#		BIT 9		DESCENT ENGINE DISABLED BY CREW
+#		BIT 9		DESCENT ENGINE GIMBALS DISABLED BY CREW
 #		BIT 10		APPARENT DESCENT ENGINE GIMBAL FAILURE
 #		BIT 14		INDICATES PROCEED KEY IS DEPRESSED
 
```

Comment:
Line below, 

- `CHAN33;` instead of `CHAN33:` (semicolon)
- Single space before `BITS 15-11` instead of double space